File: 6_api_call.js
/*
You can use python for this task if you prefer.
in javascript:
1) using fetch function, write code to fetch data from "https://fakestoreapiserver.reactbd.com/users"
2) and print the values for each user to terminal
*/

async function getData(url) {

    try {
      const response = await fetch(url);
      if (!response.ok) {
        throw new Error(`Response status: ${response.status}`);
      }

      const users = await response.json();
      for (const user of users) {
        console.log(user['name'])
    }
    } catch (error) {
      console.error(error.message);
    }
  }

const url = "https://fakestoreapiserver.reactbd.com/users";
getData(url)

File: 6_api_call.py
"""
You can use python for this task if you prefer.
in javascript:
1) using fetch function, write code to fetch data from "https://fakestoreapiserver.reactbd.com/users"
2) and print the values for each user to terminal
"""


import requests


API_ENDPOINT = 'https://fakestoreapiserver.reactbd.com/users'

response = requests.get(API_ENDPOINT)

if response.status_code == 200:
    users = response.json()

    for user in users:
        name = user['name']
        print(name)
